File: go/iq/newsletter~get~my_addons.go
// BuildNewslettersMyAddOns builds <iq type="get" xmlns="newsletter">. Replies with one of: Success, ClientError, ServerError.
func BuildNewslettersMyAddOns(limit int, jid types.JID) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": "s.whatsapp.net", "xmlns": "newsletter", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "my_addons", Attrs: waBinary.Attrs{"limit": limit, "jid": jid}},
		},
	}
}

File: ts/iq/newsletter~get~my_addons.ts
/** <iq type="get" xmlns="newsletter">. Replies with one of: Success, ClientError, ServerError. */
export function buildNewslettersMyAddOns(limit: number, jid?: string): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'newsletter', type: 'get' }, content: [
      { tag: 'my_addons', attrs: { limit: limit, jid: jid } },
    ] }
}